cmake/SetupGrpc.cmake

```diff
@@ -1,4 +1,6 @@
-find_package(gRPC QUIET)
+if(NOT USERVER_FEATURE_DONT_SEARCH_FOR_SYSTEM_GRPC_CMAKE)
```

Comment:
on my opinion, inverse flags (NOT_*) are bad
cause
- you have to inverse them in the mind to understand what actually will happened
- they produce strange statements like `if not NOT_*`

Any reasons why not to do `FOO_ENABLED` with default `1`?

Reply:
This is 100% true. Thanks.